packages/compass-e2e-tests/helpers/commands/close-settings-modal.ts

```diff
@@ -11,7 +11,6 @@ export async function closeSettingsModal(
   const settingsModalElement = await browser.$(Selectors.SettingsModal);
 
   await settingsModalElement.waitForDisplayed();
-  await browser.screenshot('settings-modal.png');
```

Comment:
what were these screenshots used for?

Reply:
Anything named modal: I worked on the modals 22 months ago and just screenshotted them all using CI so we can look at before vs after. Everything else: debugging flakes that are long gone.

Reply:
From the PR description:

> Drive-by: I removed a bunch of often two-year-old debug screenshot code because there are just so many screenshots cluttering the output when you're looking for an error screenshot.

Reply:
ah, sorry, somehow I missed that comment